src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -133,7 +133,7 @@ const getWebhookSecret = async (uuid?: string): Promise<{ webhookSecret: string,
 		if (!gitHubServerApp) {
 			throw new Error(`GitHub app not found for uuid ${uuid}`);
 		}
-		const installation: Installation = await Installation.findByPk(gitHubServerApp.installationId);
+		const installation: Installation | null = await Installation.findByPk(gitHubServerApp.installationId);
```

Comment:
Same here, let's use `!` instead because from the context we know that there must be an installation (cause we have the instance of the server app and they are linked by a foreign key)

Reply:
Same deal here, anywhere we call `findByPk` TS now complains about type null not being assignable to X.